README.adoc

```diff
@@ -138,21 +138,21 @@ include::finish/src/main/liberty/config/server.xml[tags=**]
 ---- 
 
 SystemReadinessCheck.java
-[source, java, linenums, role='code_column']
+[source, java, linenums, role='code_column hide_tags=copyright tags=SystemReadinessCheck']
 ----
-include::finish/src/main/java/io/openliberty/guides/system/SystemReadinessCheck.java[tags=SystemReadinessCheck]
+include::finish/src/main/java/io/openliberty/guides/system/SystemReadinessCheck.java[]
 ----
 
-The [hotspot=8 file=0]`@Readiness` annotation indicates that this particular bean is a readiness health check procedure. 
-By pairing this annotation with the [hotspot=9]`ApplicationScoped` context from the Contexts and 
+The [hotspot=Readiness file=0]`@Readiness` annotation indicates that this particular bean is a readiness health check procedure. 
+By pairing this annotation with the [hotspot=ApplicationScoped]`ApplicationScoped` context from the Contexts and 
 Dependency Injections API, the bean is discovered automatically when the http://localhost:9080/health[http://localhost:9080/health^] 
 endpoint receives a request.
 
-The [hotspot=12-20]`call()` method is used to return the health status of a particular service. 
-In this case, you are simply checking if the server name is [hotspot=13]`defaultServer` and 
-returning [hotspot=18-19]`UP` if it is, and [hotspot=14-16]`DOWN` otherwise. The 
-[hotspot=14]`HealthCheckResponse.named()` method is used to indicate what service the health 
-check is done for. Overall, this is a very simple implementation of the [hotspot=12-20]`call()` 
+The [hotspot=HealthCheckResponse]`call()` method is used to return the health status of a particular service. 
```

Comment:
The `call()` hotspot HealthCheckResponse does not highlight as the master site. The master highlight the whole `call()` method.

Reply:
Updated.

Reply:
The second `call()` hotspot does not work.

Reply:
Fixed.